crates/core/src/factories/core_fetch.rs

```diff
@@ -36,24 +38,38 @@
 //!   from the set of requests if it is received in the meantime, and thus prevent a redundant
 //!   fetch request.
 //!
+//! #### Responses
+//!
+//! Similarly to fetch requests, a channel serves as a queue for responses to fetch requests. The queue
+//! has the following properties:
+//! - Simple queue which processes items in the order of the incoming requests.
+//! - Requests consist of a list of requested op ids and the requesting agent id.
+//! - Attempts to look up the op in the data store and send response are done once.
+//! - Requests for data that the remote doesn't hold should be logged.
```

Comment:
This looks like it needs a second look? Is this "requests for data we don't have are logged"?

Reply:
Yes, looked twice now.